docs/guides/security-dev-productivity/hipaa-compliance.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Setting up HIPAA-compliant ngrok services
+sidebar_label: HIPAA Compliance
+description: Read ngrok's recommendations for setting up ngrok services in a HIPAA-compliant manner.
+---
+
+The Health Insurance Portability and Accountability Act (HIPAA) is the US federal law enacted to protect patient health information.
+The law sets stringent standards in order to secure electronic protected health information (ePHI/PHI).
+
+This page contains ngrok's recommendations for setting up ngrok services in a HIPAA-compliant manner.
+
+:::warning
+These recommendations from ngrok's team do not constitute legal advice.
+Please consult your own legal and engineering teams to ensure HIPAA compliance.
+:::
+
+## Shared responsibility model
+
+ngrok operates with a shared responsibility model.
+There are many safeguards that we put in place to protect our customers, and there are steps our customers must take to remain compliant with HIPAA.
+
+We're responsible for providing you, the customer, with all the information you need to use ngrok in a compliant manner, and how to configure the ngrok platform to remain compliant.
+You are responsible for ensuring your use case is compliant and configuring ngrok correctly to ensure compliance.
+
+## Compliant use cases
+
+ngrok is HIPAA-compliant for use cases where PHI is stored within a packet payload.
+You are responsible for ensuring that PHI is only present within the packet payload.
+ngrok won't store this data in HIPAA workloads but we do store other, non-PHI, data.
+See [Data at ngrok](https://ngrok.com/blog-post/data-at-ngrok) for more details on what data ngrok stores.
+
+ngrok account user information, ngrok account billing information, and packet headers should not be considered PHI within any use cases.
+
+## Customer safeguards
```

Comment:
Other safeguards to consider:
- Using a reserved domain
- Store auth tokens securely (rotate periodically?)
- Apply ACLs to your ngrok for least priv
- Regularly review logs for unauthorized access and anomolies

Reply:
Ensure TLS is enforced for client -> server